### PR TITLE
bsc#1046413:  Add IP configuration setup for libvirt-lxc

### PR DESCRIPTION
**Pull request by taroth21**
This needs review by the bug reporter - do NOT merge yet!

@cbosdo : Please review, as I'm not familiar with LXC and had to guess some details...

For a draft PDF version of the chapter, please see my Export directory (subdir ```/cbosdonnat```)

### CODE REVIEW

File: xml/lxc.xml [hunk 1]
```diff
@@ -163,6 +163,142 @@ Michal Svec (PM)
...
+    guest. However, some hypervisors or network device types will simply ignore
+    them or only use the first one.
+   </para>
+ <step>
+  <para>
+   <remark>taroth 2017-08-25: from the information on the Web, I had no idea
+   which XML file to edit -is the following correct?</remark>
+   Edit the container XML configuration using virsh:
+  </para>
+  <screen>virsh -c lxc:/// edit <replaceable>MYCONTAINER</replaceable></screen>
```
Comment (cbosdo) [inline xml/lxc.xml]: Yes it is correct

File: xml/lxc.xml [hunk 2]
```diff
@@ -163,6 +163,142 @@ Michal Svec (PM)
...
+ </step>
+ <step>
+  <para>
+   Save the changes and exit the editor.
+  </para>
+ </step>
+</procedure>
+  <para>
+   <remark>taroth 2017-08-25: is the following warning still correct and needed?</remark>
+  </para>
```
Comment (cbosdo) [inline xml/lxc.xml]: The ifcfg- part of it is still true as long as bsc#883349 is not fixed.
FWICT the IPV6 part of the warning could be remove now.

File: xml/lxc.xml (hunk 1 above)
Comment (taroth21) [reply xml/lxc.xml]: Great, thanks, removing remark.

File: xml/lxc.xml (hunk 2 above)
Comment (taroth21) [reply xml/lxc.xml]: Ok, removed the second sentence and downgraded from 'Warning' to 'Note'.

**Comment by taroth21** [comment]
This has been merged into develop with https://github.com/SUSE/doc-sle/commit/f5cf2d73

**Pull request closed without merging**